routes/cube_routes.js

```diff
@@ -1051,10 +1051,9 @@ router.post('/importcubetutor/:id', ensureAuth, body('cubeid').toInt(), flashVal
         blogpost: blogpost.toObject(),
       };
       return res.render('cube/bulk_upload', {
-        reactHTML:
-          NODE_ENV === 'production'
```

Comment:
These conditional seemed redundant since they duplicate this block:

https://github.com/dekkerglen/CubeCobra/blob/2728c61e0113f0a89bde8cdb2b240eeb77b53f99/routes/cube_routes.js#L64-L69

And they were confusing while i was debugging, so i cleaned them up!